cli/determined_cli/model.py

```diff
@@ -169,7 +170,7 @@ def register_version(args: Namespace) -> None:
                 is_default=True,
             ),
             Cmd(
-                "register_version",
+                "register-version",
```

Comment:
question: should we make this `register-model-verion`?

Reply:
The CLI command already has `model` in it (`det model register-version`), so personally I think `register-model-version` would be redundant.